dotnet/src/Connectors/Connectors.UnitTests/Memory/MemoryStorageTestBase.cs

```diff
@@ -0,0 +1,696 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel.Memory;
+using Xunit;
+
+namespace SemanticKernel.Connectors.UnitTests.Memory;
+public abstract class MemoryStorageTestBase
```

Comment:
Extra space needed :)
```suggestion

public abstract class MemoryStorageTestBase
```